assembly_knative-OCP-311.adoc

```diff
@@ -0,0 +1,16 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+
+[id='knative-ocp-311_{context}']
+= Knative on an OpenShift cluster 3.11
+
+// Knative on OpenShift cluster overview
+include::con_knative-OCP-311.adoc[leveloffset=+1]
+
+// Installing Knative on a Minishift cluster using `install-on-minishift.sh`

```

Comment:
OCP 3.11 and Minishift 1.25.0 are two different sets of instructions for installing Knative, which is why they had 3 different .md files in the previous structure. Minishift needs to be pulled into it's own section and references to it removed from 3.11 and 4.0 modules.